Mathlib/NumberTheory/EllipticDivisibilitySequence.lean

```diff
@@ -14,10 +14,10 @@ This file defines the type of an elliptic divisibility sequence (EDS) and a few
 
 ## Mathematical background
 
-Let $R$ be a commutative ring. An elliptic sequence is a sequence $W : \mathbb{Z} \to R$ satisfying
-$$ W(m + n)W(m - n)W(r)^2 = W(m + r)W(m - r)W(n)^2 - W(n + r)W(n - r)W(m)^2, $$
-for any $m, n, r \in \mathbb{Z}$. A divisibility sequence is a sequence $W : \mathbb{Z} \to R$
-satisfying $W(m) \mid W(n)$ for any $m, n \in \mathbb{Z}$ such that $m \mid n$.
+Let `R` be a commutative ring. An elliptic sequence is a sequence `W : ℤ → R` satisfying
+`W(m + n)W(m - n)W(r)² = W(m + r)W(m - r)W(n)² - W(n + r)W(n - r)W(m)²`
+for any `m, n, r ∈ ℤ`. A divisibility sequence is a sequence `W : ℤ → R` satisfying `W(m) ∣ W(n)`
+for any `m, n ∈ ℤ` such that `m ∣ n`.
```

Comment:
And what is an "elliptic divisibility sequence"? One that is simultaneously elliptic and a divisibility one?

Reply:
Yeah, I can add this.